server/src/schema.js

```diff
@@ -137,17 +157,40 @@ const typeDefs = gql`
         participants: [User]!
         maxParticipantNum: Int
     }
-
-    # Every event can have multiple tags.
-    # Tags are predefined by ours(developers)
-    # and used by the event host to categorize his event.
-    # So we are able to categorize events by tags.
-    type Tag {
+    
+    type EventBookClub implements Event{
         id: ID!
-        name: String!
-        events: [Event]!
+        host: User!
+        thumbnailUrl: String
+        creationTime: DateTime!
+        # When the schedule of the event is updated,
+        # lastUpdatedTime of the event also need to be updated.
```

Comment:
this can move to above line. Each line can contain up to 80 characters

Reply:
But, the two sentences are total 98 characters(exclude comment sign and previous blank).
Should i concatenate two lines?

Reply:
> this can move to above line. Each line can contain up to 80 characters

But, the two sentences are total 98 characters(exclude comment sign and previous blank).
Should i concatenate two lines?